website/content/docs/contribution.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Contribution"
+description: "Contributing Guidelines"
+
+weight: 5
+---
+
+First off, thanks for taking the time to contribute!
+
+The following is a set of guidelines for contributing to `Bench-routes`. These are mostly guidelines, not rules.
+Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Code of Conduct
+
+This project and everyone participates in it is governed by this [Code of Conduct](https://github.com/bench-routes/bench-routes/blob/master/CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code. Please report unacceptable behavior to [**benchroutes@gmail.com**](mailto:benchroutes@gmail.com).
+
+## How Can I Contribute?
+
+#### Contribution in code
+
+We are excited to receive contributions from you in form of code.
+
+Unsure where to begin contributing? You can start by looking through these [`beginner`](https://github.com/bench-routes/bench-routes/issues?q=is%3Aissue+is%3Aopen+label%3Afeature) and [`good-first-issue`](https://github.com/bench-routes/bench-routes/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) issues:
+
+* Beginner issues - issues which should only require a few lines of code, and a test or two.
+
+#### Pull Request Process
+
+1. Make a new branch for each feature/pull request. 
+2. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+3. Ensure that the code follows the linting standards. `eslint`, `tslint` and `prettier` are used for the TypeScript files and `golint` is used for the Golang files. Use the command `make fix` to format the code.
```

Comment:
I dont remember here, so will it be possible for you to please verify if we use all the three ways in the UI? The eslint`, `tslint` and `prettier`.

Reply:
yes, we are using all the three ways.